components/MDX/Image.tsx

```diff
@@ -39,7 +39,11 @@ export type FigureProps = ImageProps & {
 };
 
 export const Figure = ({ children, ...rest }: FigureProps) => {
-  const image = Children.only<ImageComponent>(children);
+  if (Children.count(children)) {
```

Comment:
While technically valid I prefer when conditionals evaluate to a boolean.

```suggestion
  if (Children.count(children) > 0) {
```